docs/resources/deployment.md

```diff
@@ -328,6 +329,8 @@ Required:
 
 Optional:
 
+- `instance_configuration_id` (String) Computed Instance Configuration ID of the topology element
```

Comment:
IMO it doesn't make sense to mention "Computed" - it should be clear from the schema. The code still contains similar passages but we should change that.

Reply:
Updated in https://github.com/elastic/terraform-provider-ec/pull/755/commits/7873fee0d571c5c6ecb8ce49326fbc4f31aeea21.